src/main/java/seedu/address/model/person/patient/Status.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.model.person.patient;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Patient's hospitalization status in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidStatus(String)}
+ */
+public class Status {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Status should be one of the following: Inpatient, Outpatient, Observation, Emergency Department,"
+                    + " Intensive Care Unit, Transitional Care";
+
+    public static final String VALIDATION_REGEX = "(?i)^(Inpatient|Outpatient|Observation|Emergency Department"
+            + "|Intensive Care Unit|Transitional Care)$";
+
+    private final String status;
+
+    /**
+     * Constructs a {@code Status}.
+     *
+     * @param status A valid hospitalization status.
+     */
+    public Status(String status) {
+        requireNonNull(status);
+        this.status = status.substring(0, 1).toUpperCase() + status.substring(1).toLowerCase();
```

Comment:
Should the capitalization only be for the first letter of the string or every word?